frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -57,26 +58,40 @@ export class ImportWizard extends Wizard {
     // TODO: This is horrible, but unfortunately no better way to dynamically extract only device names
     // without using ElementArrayFinder, which on the other hand may cause NoStaleElement Exceptions
     importedNICs = importedNICs.filter((_, i) => i % 3 === 0);
+    return importedNICs;
+  }
 
+  async editExistingNic(NIC: Network) {
     const NICDialog = new NetworkInterfaceDialog();
-    await asyncForEach(importedNICs, async (NIC) => {
-      await clickKebabAction(NIC.name, 'Edit');
-      await waitForNoLoaders();
-      const networks = await NICDialog.getNetworks();
-      if (networks.length > 0) {
-        await NICDialog.selectNetwork(networks[networks.length - 1]);
-      } else {
-        throw Error('No available networks to assign imported NICs');
+    await clickKebabAction(NIC.name, 'Edit');
+    await waitForNoLoaders();
+    const networks = await NICDialog.getNetworks();
```

Comment:
this logic for selecting the first available network probably doesn't make sense anymore, since we're passing the `NIC` which holds the desired state. Let's use that instead

Reply:
Please elaborate what is your idea